addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -156,14 +156,14 @@ odoo.define('website_slides.quiz', function (require) {
                         self._updateProgressbar();
                         $('#check-'+self.slide.id).replaceWith($('<i class="check-done o_wslides_slide_completed fa fa-check-circle"></i>'));
                         self.slide.done = true;
-                        self._renderSuccessModal();
+                        self._renderSuccessModal(data);
```

Comment:
I would only give to the modal the data really used in the template : points and user_karma.